### ISSUE TITLE
[Design][A11Y][Text Input] Outline contrast too low

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1058]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS 26

### App version

0.2.0

### App build number

1760480434

### App build type

beta

### Describe the issue

Outline contrast on text input is not contrasted enough:

- on white background, outline is grey #C6C6C6 which leads to a 1.7:1 contrast ratio instead of 3:1 minimum
- on dark background, outline is grey #454545 which leads to a 1.9:1 contrast ratio instead of 3:1 minimum

<img width="541" height="315" alt="Image" src="https://github.com/user-attachments/assets/d11a0537-3ce4-4208-8bc8-4e681daf9c5c" />

### Expected Behavior

Outline on text input should reach 3:1 contrast ratio minimum
For example, prefer:
- white outline on dark
- black outline on white

### Accessibility settings

none
